Iterable-iOS-SDK/IterableConstants.h

```diff
@@ -29,6 +30,8 @@ extern NSString *const ITBL_KEY_SEND_AT;
 extern NSString *const ITBL_KEY_TOKEN;
 extern NSString *const ITBL_KEY_TEMPLATE_ID;
 extern NSString *const ITBL_KEY_TOTAL;
+extern NSString *const ITBL_KEY_UNSUB_CHANNEL;
```

Comment:
do all these keys really need to be public/in the `.h`? I feel like they're internal to our API and don't particularly need to be exposed... or is this just the way it is in obj-c? can't remember...